updated build section in README.md (#407)
fredrikhgrelland/vagrant-hashistack: README.md

make install
```

> ## Build the vagrant box

Reviewer: Remember to update the `Content` section `3. [Build](#build)` :point_right: `3. [Build the vagrant box](#build-the-vagrant-box)`

Author: Thanks!

> ## Build

Reviewer: ```suggestion
## Build the vagrant box
```
Or something more descriptive?

Author: I like your suggestion

Make build will build a vagrant box based on [fredrikhgrelland/bionic64-ansible-docker](https://app.vagrantup.com/fredrikhgrelland/boxes/bionic64-ansible-docker). The packaged box will be locally available at ´packer/output-hashistack/package.box´
```text
make build
```

> Make dev will build a vagrant box as ``make build``, but without running the tests

Reviewer 2: not necessary formating 
```suggestion
Make dev will build a vagrant box as `make build`, but without running the tests
```

```text
make dev
```

> Make test runs through your ansible playbook test, you can give the box different attributes by adding them in the``.env`` file under the ```test``` directory 

Reviewer 2: not necessary formating 
```suggestion
Make test runs through your ansible playbook test, you can give the box different attributes by adding them in the`.env` file under the `test/` directory 
```

```text
make test
```

> This will destroy and take down your box if there is any

Reviewer: ```suggestion
The following command will destroy and take down your box if there is any:
```

> ```text
> make build
> ```
> Make build will build a vagrant box based on [fredrikhgrelland/bionic64-ansible-docker](https://app.vagrantup.com/fredrikhgrelland/boxes/bionic64-ansible-docker). The packaged box will be locally available at ´packer/output-hashistack/package.box´

Reviewer: ```suggestion
Make build will build a vagrant box based on [fredrikhgrelland/bionic64-ansible-docker](https://app.vagrantup.com/fredrikhgrelland/boxes/bionic64-ansible-docker). The packaged box will be locally available at ´packer/output-hashistack/package.box´

```text
make build
```

```
I would prefer that you explain the command before mention how to run it.

```text
make dev
```
Make dev will build a vagrant box as ``make build``, but without running the tests

```text
make test

> ```

Reviewer 3: Explain what `make test` does? Or maybe I'm missing something?

> ```

Reviewer: Same with those. Explanation then execution

> 
> ```text
> make clean
> ```
> This will destroy and take down your box if there is any

Reviewer: ```suggestion
This will destroy and take down your box if there is any

```text
make clean
```
```
